src/earn/PoolCard.tsx

```diff
@@ -47,143 +45,106 @@ export default function PoolCard({
+        .reduce(
+          (acc, earnItem) =>
+            acc.plus(
+              new BigNumber(earnItem.amount).times(
+                allTokens[earnItem.tokenId]?.priceUsd ?? new BigNumber(0)
+              )
+            ),
+          new BigNumber(0)
+        )
         .toFixed(2),
     [earningItems]
   )
-  const rewardPercentageString = `${rewardPercentage}%`
+  const rewardAmountInFiat =
+    useDollarsToLocalAmount(new BigNumber(rewardAmountInUsd)) ?? new BigNumber(0)
+
+  const poolBalanceString = useMemo(
+    () =>
+      `${localCurrencySymbol}${poolBalanceInFiat ? formatValueToDisplay(poolBalanceInFiat.plus(rewardAmountInFiat)) : '--'}`,
+    [localCurrencySymbol, poolBalanceInFiat, rewardAmountInFiat]
+  )
+
+  const tvlInFiat = useDollarsToLocalAmount(
+    tvl ? new BigNumber(tvl).times(new BigNumber(priceUsd)) : null
+  )
+  const tvlString = useMemo(() => {
+    return `${localCurrencySymbol}${tvlInFiat ? formatValueToDisplay(tvlInFiat) : '--'}`
+  }, [localCurrencySymbol, tvlInFiat])
 
   const totalYieldRate = new BigNumber(
     yieldRates.reduce((acc, yieldRate) => acc + yieldRate.percentage, 0)
   ).toFixed(2)
 
+  const onPress = () => {
+    AppAnalytics.track(EarnEvents.earn_pool_card_press, {
+      poolId: positionId,
+      depositTokenId,
+      networkId,
+      tokenAmount: balance,
+      providerId: appId,
+    })
+    // TODO(ACT-1321): Navigate to pool details screen
+  }
+
   return (
-    <View style={styles.card} testID={testID}>
-      <View style={styles.titleRow}>
-        {tokensInfo.map((token, index) => (
-          <TokenIcon
-            key={index}
-            token={token}
-            viewStyle={index > 0 ? { marginLeft: -8, zIndex: -index } : {}}
-          />
-        ))}
-        <View style={styles.titleTextContainer}>
-          <Text style={styles.titleTokens}>
-            {tokensInfo.map((token) => token.symbol).join(' / ')}
-          </Text>
-          <Text style={styles.titleNetwork}>
-            {t('earnFlow.poolCard.onNetwork', { networkName: NETWORK_NAMES[networkId] })}
-          </Text>
-        </View>
-      </View>
-      <View style={styles.keyValueContainer}>
-        <View style={styles.keyValueRow}>
-          <Text style={styles.keyText}>{t('earnFlow.poolCard.rate')}</Text>
-          <Text style={styles.valueTextBold}>
-            {t('earnFlow.poolCard.apy', {
-              apy: totalYieldRate,
-            })}
-          </Text>
-        </View>
-        <View style={styles.keyValueRow}>
-          <Text style={styles.keyText}>{t('earnFlow.poolCard.reward')}</Text>
-          <Text style={styles.valueText}>{rewardPercentageString}</Text>
-        </View>
-        <View style={styles.keyValueRow}>
-          <Text style={styles.keyText}>{t('earnFlow.poolCard.tvl')}</Text>
-          <Text style={styles.valueText}>{`$${new BigNumber(tvl ?? 0).toFormat()}`}</Text>
+    <Touchable style={styles.card} testID={testID} onPress={onPress}>
```

Comment:
The borderRadius should be specified on the touchable to prevent the ripple effect from going outside the component on tap on Android.

```suggestion
    <Touchable borderRadius={12} style={styles.card} testID={testID} onPress={onPress}>
```

Reply:
added in [0ced40b](https://github.com/valora-inc/wallet/pull/5787/commits/0ced40bb7aebc8de50b2af82141145ff9bf64ef2)